kustomize/commands/openapi/fetch/fetch.go

```diff
@@ -21,10 +24,20 @@ in the user's kubeconfig`,
 			printSchema(w)
 		},
 	}
-	return &infoCmd
+	fetchCmd.Flags().StringVar(
+		&format,
+		"format",
+		"json",
+		"Specify format for fetched schema ('json' or 'yaml')")
+	return &fetchCmd
 }
 
 func printSchema(w io.Writer) {
+	if format != "json" && format != "yaml" {
+		fmt.Fprintln(w, "format must be either 'json' or 'yaml'")
```

Comment:
Can we print this to stderr and have the command exit non-zero when given invalid flag values?

Reply:
done